department-of-veterans-affairs/va.gov-cms
# VACMS-9670 update error message text to plainer language

## Description

Closes #9670.

## Testing done

Entered an absolute link in a rich text field, saw that the validation error message was updated.

## Screenshots
![Screen Shot 2022-07-15 at 9 25 20 AM](https://user-images.githubusercontent.com/11279744/179267744-31b58d1e-2d42-409a-896b-c505b5f7e751.png)


## QA steps

As any user, enter an absolute link in a rich text field. `https://prod.cms.va.gov/node/5` was the example link I used.

Attempt to save the node, see the validation message is updated with the new copy.

### Definition of Done


### Select Team for PR review
